src/containers/header-profile/header-profile.js

```diff
@@ -89,7 +93,14 @@ const HeaderProfile = ({ fromSideBar }) => {
     {
       value: PROFILE_OPTIONS_VALUES[language].logIn,
       icon: <ExitToAppIcon />,
-      clickHandler: () => handleRedirect('/login')
+      clickHandler: () => {
+        const pathName = history.location.pathname;
+        const returnPath =
+          (pathName === pathToRegister || pathName === pathToLogin ? '/' : pathName) +
+          history.location.search;
+        sessionStorage.setItem(RETURN_PAGE, returnPath);
+        handleRedirect('/login');
```

Comment:
please remove magic string and use const pathToLogin from configs

Reply:
Fixed, Thank you